crates/ui/src/components/icon.rs

```diff
@@ -346,6 +347,7 @@ impl IconName {
             IconName::DebugRestart => "icons/debug-restart.svg",
             IconName::DebugStop => "icons/debug-stop.svg",
             IconName::DebugDisconnect => "icons/debug-disconnect.svg",
+            IconName::DebugLogBreakpoint => "icons/debug-log-breakpoint.svg",
```

Comment:
Can you change this? This is needed for mergin this PR.
```suggestion
            IconName::DebugLogBreakpoint => "icons/debug_logbreakpoint.svg",
```

Reply:
Thanks for the heads up!